Library/Homebrew/github_packages.rb

```diff
@@ -205,10 +205,7 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, dry_run:)
       "org.opencontainers.image.url"           => bottle_hash["formula"]["homepage"],
       "org.opencontainers.image.vendor"        => org,
       "org.opencontainers.image.version"       => version,
-    }
-    formula_annotations_hash.each do |key, value|
-      formula_annotations_hash.delete(key) if value.blank?
-    end
+    }.compact
```

Comment:
This behaviour change is a bit undesirable as it'll allow empty strings whereas the previous version didn't.

Reply:
Ah, I missed that distinction. I'll open a PR to fix it.

Reply:
I see you already did. I opened this PR https://github.com/Homebrew/brew/pull/11036 with a stylistic improvement IMO to use `reject v.blank?` to remove empty hash values.